py/torch_tensorrt/dynamo/lowering/_decompositions.py

```diff
@@ -162,6 +166,46 @@ def var_decomposition(
     return variance
 
 
+@register_torch_trt_decomposition(
+    torch.ops.aten.slice_scatter.default, registry=TORCH_TRT_DECOMPOSITIONS
+)
+def slice_scatter_decomposition(
+    input_tensor: torch.Tensor,
+    src_tensor: torch.Tensor,
+    dim: int,
+    start: Optional[int],
+    end: Optional[int],
+    step: int,
+):
+    dim_size = input_tensor.shape[dim]
+    start = get_positive_dim(start, input_tensor.shape)
+    end = get_positive_dim(end, input_tensor.shape)
+
+    src_dim = src_tensor.shape
+    step_dim = (end - start) // step
+    end_dim = end
+    if step_dim > src_dim[dim]:
+        end_dim = src_dim[dim]
+    else:
+        indices = torch.Tensor(torch.arange(0, step_dim))
+        indices = indices.to(torch.int32)
+        src = torch.index_select(src, dim, indices)
+
+    if start == 0 and end == dim_size and step == 0:
```

Comment:
Should this be `step == 1`?

Reply:
I think this should be `step == 0` since `step == 1` would result in tensors being inserted in source tensor at step 1 interval.